main/src/ca/uwaterloo/flix/language/phase/Inliner.scala

```diff
@@ -305,4 +317,25 @@ object Inliner {
 
     case Expression.MatchError(_, _) => exp0
   }
+
+  /**
+   * returns `true` if `exp0` is considered a trivial expression
```

Comment:
I would like a comment that explains:

- What is a trivial expression
- What is it used for.